### DISCUSSION TITLE
How import node image in Next.js?

### DISCUSSION THREAD

**Discussion by kirillleogky** [vercel/next.js#44106] [Help]
<img width="370" alt="Screen Shot 2022-12-17 at 3 02 06 PM" src="https://user-images.githubusercontent.com/47118725/208240859-bd038e69-be1f-42f3-b6b7-9eee6a2c7079.png">

`import GamingPad from 'public/icons/gaming-pad.svg';`
<img width="370" alt="Screen Shot 2022-12-17 at 3 02 44 PM" src="https://user-images.githubusercontent.com/47118725/208240878-57dbc625-2dea-437e-bbfc-ef776b32eda1.png">

I see this object:
```
{
  src: '/_next/static/media/gaming-pad.d149579e.svg',
  height: 32,
  width: 32
}

```

How I can change my import images from objects to node?
I'm using mui for styles and have problem:
<img width="827" alt="Screen Shot 2022-12-17 at 3 04 39 PM" src="https://user-images.githubusercontent.com/47118725/208240976-6b629356-3b06-46d1-8f5e-f1fc636f117f.png">

<br>
<br>
<br>

`next.config.js`
```
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
  env: {
    ADMIN_SECRET: process.env.ADMIN_SECRET,
  },
};

module.exports = nextConfig;

```

**Comment by wwwdepot** [comment]
To import svg as a component. Use SVGR. https://react-svgr.com/docs/next/

MUI guide https://mui.com/material-ui/icons/#component-prop
